File: app/src/androidTest/java/org/tasks/makers/Maker.java
package org.tasks.makers;

import static com.natpryce.makeiteasy.MakeItEasy.a;

import com.natpryce.makeiteasy.Instantiator;
import com.natpryce.makeiteasy.PropertyValue;

class Maker {

  @SuppressWarnings("unchecked")
  public static <T> T make(
      Instantiator<T> instantiator, PropertyValue<? super T, ?>... properties) {
    return com.natpryce.makeiteasy.MakeItEasy.make(a(instantiator, properties));
  }
}

File: app/src/sharedTest/java/org/tasks/makers/Maker.kt
package org.tasks.makers

import com.natpryce.makeiteasy.Instantiator
import com.natpryce.makeiteasy.MakeItEasy
import com.natpryce.makeiteasy.PropertyValue

internal object Maker {
    fun <T> make(instantiator: Instantiator<T>, vararg properties: PropertyValue<in T, *>?): T {
        return MakeItEasy.make(MakeItEasy.a(instantiator, *properties))
    }
}